app/services/batched_remove_status_service.rb

```diff
@@ -73,10 +73,12 @@ def unpush_from_public_timelines(status)
     redis.pipelined do
       redis.publish('timeline:public', payload)
       redis.publish('timeline:public:local', payload) if status.local?
+      redis.publish('timeline:public:remote', payload) unless status.local?
 
       if status.media_attachments.any?
         redis.publish('timeline:public:media', payload)
         redis.publish('timeline:public:local:media', payload) if status.local?
+        redis.publish('timeline:public:remote:media', payload) unless status.local?
```

Comment:
Maybe best to rework these into if/else instead of if/unless!

Reply:
I got it.